tests/extras/extensions/test_ipython.py

```diff
@@ -62,18 +56,18 @@ def my_register_pipeline():
         mocker.patch(
             "kedro.framework.startup.bootstrap_project", return_value=fake_metadata
         )
-        mock_line_magic = mocker.MagicMock()
+        mock_line_magic = mocker.Mock()
```

Comment:
Just curious is this necessary? Or we just prefer `Mock` in general. I think we use `MagicMock` quite a lot but without actually needing it in some tests.

Reply:
Good question - I did a quick survey of our use of `MagicMock` vs. `Mock` and found that nearly always it's `Mock` apart from in this file. There's no need for the extra capability of `MagicMock` here so I changed them all.